src/components/base/view.js

```diff
@@ -160,6 +163,21 @@ genotet.View.prototype.headerText = function(headerText) {
 };
 
 /**
+ * Sets header extra information of the view. If null, return the current
+ * header.
+ * @param {?string} extraInfo Extra information in the header.
+ * @return {?string}
```

Comment:
Maybe we can return always return a string. If it is set info operation, then return extraInfo itself. Same for headerText().